app/javascript/app/components/modal-metadata/modal-metadata-reducers.js

```diff
@@ -20,9 +20,9 @@ const setLoading = (state, loading) => ({ ...state, loading });
 const setLoaded = (state, loaded) => ({ ...state, loaded });
 const setData = (state, { slugs, data }) => {
   let slugData = {};
+  const leData = data[0].data;
```

Comment:
Also here @Bluesmile82 . Not sure if this is the best way to do it, but I didn't want to go too far into figuring that out. =)